backends-velox/src/main/scala/io/glutenproject/execution/VeloxFilterExecTransformer.scala

```diff
@@ -32,7 +32,7 @@ import org.apache.spark.sql.catalyst.expressions.{And, Attribute, Expression}
 import org.apache.spark.sql.execution.SparkPlan
 
 
-case class FilterExecTransformer(condition: Expression, child: SparkPlan)
+case class VeloxFilterExecTransformer(condition: Expression, child: SparkPlan)
```

Comment:
Why do we need this change? Since this class in in `backends-velox` module, the scope could be apparant.

Reply:
A class with same name is in `gluten-core` and accessed by `backends-clickhouse`.

Reply:
Maybe we can merge that `FilterExecTransformer` into `FilterExecBaseTransformer`, and use `FilterExecBaseTransformer` in CH backend. Does that make sense? I can do this refactor if needed.

Reply:
Making sense to me, so let's use a separate patch for that change?

This patch is simply to add extra restrictions for future development. It's more or less unrelated to specific functionalities.

Reply:
OK, sure. Will open PR to do that after this change.